operators/src/processing/expression.rs

```diff
@@ -542,7 +542,7 @@ mod tests {
         let raster_tile = RasterTile2D::new_with_tile_info(
             TimeInterval::default(),
             TileInformation {
-                global_tile_position: [0, 0].into(),
+                global_tile_position: [-1, 0].into(),
```

Comment:
Why was it necessary to change this here and why do the tests still run through?

Reply:
cf other comment